packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.spec.ts

```diff
@@ -72,65 +64,4 @@ describe('LitNodeClientNodeJs', () => {
 
     expect(expiration).toContain('T');
   });
-
-  describe('normalizeParams', () => {
```

Comment:
Why remove these tests?

Reply:
They were moved to helper functions now, so these tests are redundant